transactions.go

```diff
@@ -157,8 +157,13 @@ func (t ItemTransaction) Mutate(b *Battle) {
 			Stages: 1,
 		})
 	case ItemStarfBerry:
-		// TODO: boost random stat, requires battle RNG to be available.
-	// ItemCategoryHeldItems
+		if target.CurrentHP <= target.MaxHP()/4 {
```

Comment:
The if statement is not necessary. This condition is already known to be true at this point.